app/javascript/components/ansible-repository-form/ansible-repository-form.schema.js

```diff
@@ -43,7 +43,8 @@ function createSchema(repositoryId) {
         message: __('Required'),
       },
       {
-        type: validatorTypes.URL,
+        type: validatorTypes.PATTERN,
+        pattern: /^(http:\/\/)|(https:\/\/)|(file:\/\/)|([[:alnum:]]@[[:alnum:]])|(ssh:\/\/)/,
```

Comment:
don't think alnum is right here, because you can have some special characters.  We might need a much more complicated version of URL.

Reply:
yeah the alnum was just v1 for me to test some simple cases first, this is the WIP still so i'm still adding more nuance to it now